src/components/Dashboard/CreateRoom/CreateRoom.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import axios from 'axios';
+import Button from '../../shared/Button/Button';
+
+export class CreateRoom extends React.Component {
+    createRoom() {
+        axios.get('/api/room/create')
```

Comment:
This call isn't returning anything so I don't think it should be a `get`. Looks like it should be `post` instead.